README.md

```diff
@@ -1,68 +1,69 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Hack Zero
+> Hackathon management web application <br>use in tandem with [Hack-Zero-Backend](https://github.com/hackthehack/hack-zero-v2-backend)
 
-## Available Scripts
+[![NPM Version][npm-image]][npm-url]
+[![Build Status][travis-image]][travis-url]
```

Comment:
were not using travis, you can get icons for github actions

Reply:
ah yep!